README

```diff
@@ -23,6 +23,12 @@ which can be done by using the https://developers.yubico.com/yubikey-manager-qt/
 
 Downloads for all supported operating systems are available https://developers.yubico.com/yubioath-desktop/Releases/[here].
 
+
+==== Linux
```

Comment:
Can we add this as a single sentence under System Requirements instead?
`On Linux systems, make sure the pcscd service is installed and running`.

Reply:
Sure!